### DISCUSSION TITLE
CL automatically assigned to the stable score should be System Mod

### DISCUSSION THREAD

**Discussion by shiumano** [ppy/osu#28993] [Ideas] [answered]
When playing with stable, CL is forced to be given without setting it and cannot be removed. Doesn't this sound like a TD Mod?
It seems to me that there are not a few problems caused by the CL indicating that you are playing in stable and the CL indicating that you are clinging to the past in lazer being treated as the same thing.
What I propose is to change the CL given to stable to a system mod and make the lazer one a conversion mod as it has always been.
It would also be a good idea to rename it to distinguish it from the mod that is attached to the score of stable.

**Comment by bdach** [comment, marked as answer]
No. Not happening.
